.github/workflows/build.yml

```diff
@@ -66,6 +66,23 @@ jobs:
           command: fmt
           args: --all -- --check
 
+  cargo-deny:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        checks:
+          - advisories
+          - bans licenses sources
+
+    # Prevent sudden announcement of a new advisory from failing ci:
+    continue-on-error: ${{ matrix.checks == 'advisories' }}
+
+    steps:
+    - uses: actions/checkout@v2
+    - uses: EmbarkStudios/cargo-deny-action@v1
```

Comment:
Could you pin these two by hash (`actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b` as above for `actions/checkout` and seems to be `7257a18a9c2fe3f92b85d41ae473520dff953c97` for `EmbarkStudios/cargo-deny-action` v.1.3.2 ). That will please some Google-recommended security scanner we have enabled.

Reply:
Done!